File: 프로그래머스/lv1/68644. 두 개 뽑아서 더하기/두 개 뽑아서 더하기.java
import java.util.*;

class Solution {
    public int[] solution(int[] numbers) {
        TreeSet<Integer> tree = new TreeSet<>();
        for(int i = 0; i < numbers.length - 1; i++) {
            for(int j = i + 1; j < numbers.length; j++) {
                tree.add(numbers[i] + numbers[j]);
            }
        }
        int[] answer = tree.stream().mapToInt(Integer::intValue).toArray();
        return answer;
    }
}

File: 프로그래머스/1/68644. 두 개 뽑아서 더하기/두 개 뽑아서 더하기.js
function solution(numbers) {
    var answer = new Set();
    for(let i = 0; i < numbers.length - 1; i++) {
        for(let j = i + 1; j < numbers.length; j++) {
            answer.add(numbers[i] + numbers[j]);
        }
    }
    return [...answer].sort((a, b) => a - b);
}